.github/workflows/release_bundle.yml

```diff
@@ -36,30 +36,67 @@ on:
         required: true
 
 jobs:
+  build-python-matrix:
```

Comment:
I think what the `snowflake-connector-python` issue identified is that we're using the same requirements.txt file for all versions of python. In other words, we're running a `pip freeze` into a requirements file for `py38` and then assuming those patch versions are available across all supported versions of python. Even without the missing wheel issue that we saw, this may not always be the case. Does this resolve that? Or is that unrelated?

Reply:
This is unrelated.  This is because we support different versions of python depending on the minor version of dbt we're trying to generate bundles for.

I solved that issue by [pinning in the requirements file](https://github.com/dbt-labs/dbt-core-snapshots/blob/1d0da79bc3d5bd4051be6cbbb1b39231e1f69bcc/release_creation/snapshot/requirements/v1.6.pre.requirements.txt#L12) for now.  Definitely a short term fix.  PyPi has responded on their issue so it will hopefully move forward soon for an actual fix.